adapters/kayzen/kayzentest/exemplary/banner-web.json

```diff
@@ -0,0 +1,129 @@
+{
+  "mockBidRequest": {
```

Comment:
do we really need these many test cases? we will not use web anyways.

Reply:
If we are supporting web externally, even if we use it or not we have to write testcases for them
@serveshjain/Jatin mentioned if it is easier to support both app & web do it.. So no other option